### PR TITLE
Implement create reaction UI 

### PR DESCRIPTION
**Pull request by haiyingweng**
### Overview 
Currently just use hardcoded data for title and episode

### Screenshots/GIFs

https://user-images.githubusercontent.com/48968041/156690815-5536b367-e42a-4e65-9c8b-d33116b1c1bd.mov

newer UI change: 
Screen Shot 2022-03-03 at 11 31 01 PM

### CODE REVIEW

File: Telie/Controllers/CreateReactionViewController.swift [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+
+    private func setupConstraints() {
+        let leadingTrailingPadding: CGFloat = 20
+        let verticalPadding: CGFloat = 10
+
+        titleLabel.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide).offset(verticalPadding)
+            make.leading.equalToSuperview().offset(leadingTrailingPadding)
+            make.trailing.equalTo(changeButton.snp.leading).offset(leadingTrailingPadding)
+        }
```
Comment (alanna-zhou) [inline Telie/Controllers/CreateReactionViewController.swift:162]: ohh so this is snapkit i seeee

**Review by alanna-zhou — APPROVED** [review]
y r u so cool.

**Comment by cindyh25** [comment]
could we make the button full width because we don't have image/audio stuff?

**Comment by haiyingweng** [comment]
> could we make the button full width because we don't have image/audio stuff?

oh, it is actually full width. you can see the second image in the description. Sry I made the screen recording when I didn't make that change.

**Pull request merged**